### ISSUE TITLE
Stop showing relative dates for very old files

### ISSUE THREAD

**Issue by Daniel15** [haiwen/seahub#3965]
For very old files, Seadrive used to display the timestamp in ISO8601 format (eg. `2005-01-01`). The public "shared links" do still display it this way:
![](http://ss.dan.cx/2019/08/Daniel15%27s_Files_-_Mozilla_Firefox_08-22.43.28.png)

However, after upgrading from 6.3.4 to 7.0.4, it's just showing relative timestamps for everything, which is much less useful:
![](http://ss.dan.cx/2019/08/Daniel15%27s_Files_-_Mozilla_Firefox_09-00.06.01.png)

Additionally, it does not display the full date/time on hover, so there's no way to get a more granular timestamp than "17 years ago"

Was this an intentional change? Is it possible to restore the old behaviour?

**Comment by Michael18811380328** [comment]
Actually, it is a problem. Using relative time is convenient for recent file. You can easily distinguish between old files and new files. And using absolute time is convenient for history file.
<img width="284" alt="file" src="https://user-images.githubusercontent.com/37589122/64333386-fc235300-d008-11e9-9f6d-fa2146d4f057.png">

**Comment by Daniel15** [comment]
Yeah, I agree that relative time is good for recent files, whereas absolute time is good for older files. That's exactly how Seafile 6.x behaved.
